_docker/drupal/drupal-filesystem/web/themes/custom/rhd/css/base/eclipse.css

```diff
@@ -0,0 +1 @@
+.wide.wide-hero.eclipse{background:url(/images/eclipse/eclipse_microsite_hero.jpg);background-size:cover;height:450px}
```

Comment:
@robpblake I was wondering about that. It showed up after a build in my git status, but I thought it was generated because of the changes I made. @LightGuard do I just need to remove this and update the .gitignore?